app/views/shared/_countries.slim

```diff
@@ -0,0 +1,2 @@
+javascript:
+  window.champaign.countries = #{ISO3166::Country.all.map { |c| c.instance_values['country_data_or_code'] }.to_json.html_safe }
```

Comment:
Can we try and get this significantly smaller in size? Perhaps we should also cache it? In development, it's taking 200ms to render, and it's over 500k in size!